### PR TITLE
Add common Australian items to stores & brands

### PR DESCRIPTION
**Pull request by CloCkWeRX**
For 69ae403; not sure if this is useful, or if #84 is just easier.

I haven't explored much around the common aliases from OCR, but happy to add if things like apostrophes are often mistaken.

### CODE REVIEW

File: data/ocr/regex_brands.txt [hunk 1]
```diff
@@ -23,13 +28,16 @@ Carrefour Baby||carrefour [bg]aby
 Carrefour Bio
 Carrefour Discount
 Chabrior
+Coles
```
Comment (teolemon) [inline data/ocr/regex_brands.txt]: Coleslow ? @raphael0202

Comment (raphael0202) [reply data/ocr/regex_brands.txt]: @teolemon it seems that Coles is indeed a brand: https://au.openfoodfacts.org/brand/coles

**Comment by raphael0202** [comment]
Thank you for the PR, and sorry for not replying sonner.
For brands and stores, a 'notify' list allows to be informed when automatic annotations are made with these brands/stores on Slack (on the robotoff-alert channel): https://github.com/openfoodfacts/robotoff/blob/master/data/ocr/notify_brands.txt and https://github.com/openfoodfacts/robotoff/blob/master/data/ocr/notify_stores.txt.
Some brands/stores seem a bit problematic to me (Peters, Coles, Bega, Albert's,...), as these are short (or common) strings that can easily be found in OCR texts. Can you trigger notifications for those (and other that seem problematic to you?). You can delete all current notifications for brands and stores, we had enough time to check that there were no false positive on those.

**Comment by CloCkWeRX** [comment]
I'll tweak things for the ambiguous ones shortly then

**Comment by raphael0202** [comment]
@CloCkWeRX I've implemented https://github.com/openfoodfacts/robotoff/pull/84, and those australian brand are already in the taxonomy

**Pull request closed without merging**